lib/RTCSession.js

```diff
@@ -2632,6 +2656,26 @@ module.exports = class RTCSession extends EventEmitter
       });
   }
 
+  /**
+   * Get DTMF RTCRtpSender.
+   */
+  _getDTMFRTPSender()
+  {
+    const senders = this._connection.getSenders().find((sender) =>
```

Comment:
This is not `senders` but `sender`, right?

```suggestion
    const sender = this._connection.getSenders().find((sender) =>
```

Reply:
done